src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -65,6 +65,7 @@
  */
 class FirebaseUserManager {
 
+  static final String CONFIGURATION_NOT_FOUND = "project-not-found";
```

Comment:
I find this mapping strange, however, "CONFIGURATION_NOT_FOUND" was already being mapped to "project-not-found".

I've confirmed that the server returns CONFIGURATION_NOT_FOUND if you ask it to delete a provider config ID which doesn't exist.

Reply:
Already mapped where? Looks like it's being added here.

Reply:
It was already in the map (`.put("CONFIGURATION_NOT_FOUND", "project-not-found")`), I just gave the constant a name so that it could be referenced.

Reply:
I think we ought to fix it. I don't think any of the existing APIs return this error code, so it should be safe to change it to "configuration-not-found".

Reply:
Okay! Done.